app/src/main/res/layout/list_item_transport_network.xml

```diff
@@ -11,8 +11,8 @@
 
 	<ImageView
 		android:id="@+id/logo"
-		android:layout_width="wrap_content"
-		android:layout_height="wrap_content"
+		android:layout_width="64dp"
```

Comment:
This won't look good for PNG images that we still have a few of. Better leave it as it was before.

Reply:
I'm not sure anymore, but I think this was producing different sizes which looks a bit odd. I added the value to keep a consistent text position throughout continent, country and transport network items.

How many PNG files are there? If there aren't too much, I'm ok with reverting this change here and open another PR converting the PNGs to vectorgraphics. Or at least resizing the PNGs to exactly 64 pixels width.

Reply:
Note that `dp` is not `pixels`. I wouldn't put work in enlarging PNGs. There is #238 and having the PNGs in a different size is a nice reminder of that.

Reply:
Ok, I've reverted this one.